client/auth/index.js

```diff
@@ -10,8 +10,9 @@ import config from 'config';
 import controller from './controller';
 
 export default () => {
+	page( '/oauth-login', controller.oauthLogin );
```

Comment:
Cause the desktop app records the last route so some users might land on `/oauth-login` and we want to redirect them to `/log-in`. Also I'm not sure we should simply remove oauth2. We might want to come back to it later. Anyway it should not be part of this PR imo